### PR TITLE
[WFCORE-5912] [blacklist/whitelist] Find and replace occurrences in d…

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/WFCORE-5912

### CODE REVIEW

File: remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties [hunk 1]
```diff
@@ -124,7 +124,7 @@ local-outbound-connection.property=The XNIO Options that will be used during the
 sasl=The "sasl" element contains the SASL authentication configuration for this connector.
 sasl.add=Adds the SASL authentication configuration for its connector
 sasl.remove=Removes the SASL authentication configuration for its connector
-sasl.include-mechanisms=The optional nested "include-mechanisms" element contains a whitelist of allowed SASL mechanism names. No mechanisms will be allowed which are not present in this list.
+sasl.include-mechanisms=The optional nested "include-mechanisms" element contains a allowlist of allowed SASL mechanism names. No mechanisms will be allowed which are not present in this list.
```
Comment (yersan) [inline remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties:127]: s/a allowlist/an allowlist. However, in any case, it results in redundancy.


What about if we use:

"The optional nested "include-mechanisms" element contains a list of allowed SASL mechanism names. No mechanisms will be allowed which are not present in this list"

or

"The optional nested "include-mechanisms" element contains an allowlist of SASL mechanism names. No mechanisms will be allowed which are not present in this list"


@fjuma as a native English speaker, would you mind helping us with this sentence? thanks!

**Review by yersan — CHANGES_REQUESTED** [review]

File: remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties (hunk 1 above)
Comment (fjuma) [reply remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties:127]: @yersan The first option you proposed looks good. I wonder if we could reword the second sentence to make it more clear, e.g.,

"The optional nested "include-mechanisms" element contains a list of allowed SASL mechanism names. Only mechanisms that are present in this list will be allowed."

Comment (yersan) [reply remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties:127]: Thanks, Farah. That sounds also more clear to me too, +1 to your proposal.

Comment (moulalis) [reply remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties]: Thanks @fjuma and @yersan, I've updated the PR accordingly.

**Pull request merged**